src/gatling/resources/RecordedSimulationFRApplicant_0051_request.txt

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
is this file needed? also that event_token is dynamic and will expire after a few hours
URLs should be parameterised, as should the doc-id - dynamic or parameterised?

Reply:
Parameterised values.

Reply:
Made token dynamic and parameterised values.